Projektdoku/Projektdoku.tex

```diff
@@ -0,0 +1,17 @@
+\documentclass{scrartcl}
+
+\usepackage[ngerman]{babel}
+
+\begin{document}
+	\title{Prüfungsplaner \\ Projektdokumentation}
+	\author{Gruppe 7}
+	\date{Sommersemester 2018}
+	\maketitle
+	
+	\tableofcontents
+	
+	Diese Dokument wird der Projektplan und -ablauf der Applikation "Prüfungsplaner" beschrieben. Sie wurde im Rahmen der Belegarbeit Software-Engineering 2 von entwickelt. Die Anwendung dient zur Unterstützung und Planung von Präsentationsterminen.
+	
```

Comment:
Hier (oder in der Entwicklerdoku, bin mir nicht ganz sicher) muss noch die eine Pflichtenheftänderung rein (MSSQL statt MySQL-Datenbank)